addon/services/file-manager.js

```diff
@@ -141,25 +244,65 @@ export default Ember.Service.extend({
                 headers: {
                     Authorization: `Bearer ${accessToken}`
                 },
-                success: (data) => resolve(data),
-                error: (_, __, error) => reject(error)
+                error: (_, __, error) => reject(error),
```

Comment:
It does return a deferred object, but since that has a slightly different interface from  `Ember.RSVP.Promise` (there's no `deferred.catch`), I didn't want to expose it to the outside. Wrapping it in an extra Promise seemed like a small sacrifice for consistency.

That said, I can see how it would be cleaner to use the Deferred semantics than `success` and `failure`, within the wrapping promise.